### DISCUSSION TITLE
How to use --partitions options?

### DISCUSSION THREAD

**Discussion by chthtlo** [Altinity/clickhouse-backup#570] [General]
Hi Everyone!

I want to create dump only for specifics partitions in table, but I couldn't do it.
I've used partition name from system.parts and I got empty backup:
```
# clickhouse-backup list
2022/11/25 09:45:54.270576  info SELECT value FROM `system`.`build_options` where name='VERSION_INTEGER'
2022/11/25 09:45:54.273347  info SELECT * FROM system.disks;
backup_name   1.12KiB   25/11/2022 09:45:49   local  
```

Command:
`./clickhouse-backup create backup_name -c config-test.yml --table database.table --partitions "(20221031,'execution')"`

Where is my mistake? Thanks.

**Comment by Slach** [comment]
You should use partition_id instead of partition_name
something like `--partitions="20221031_execution'`

**Reply by chthtlo** [reply]
Hi, Slach! Thanks a lot, that solved my issue, with the id partition I was able to do a backup.
